File: backend/components/metiers/threed/security/docs/guides/samples/api/api_samples.js
// api_samples.js – Exemples d’intégration API (Node.js)
function sampleApiRequest() {
  return {
    method: 'POST',
    url: '/api/secure-endpoint',
    headers: { Authorization: 'Bearer <token>' },
    body: { data: 'example' }
  };
}
module.exports = { sampleApiRequest };

File: backend/components/metiers/threed/security/docs/guides/samples/api/api_samples.py
# api_samples.py – Exemples d’intégration API (Python)
def sample_api_request():
    return {
        'method': 'POST',
        'url': '/api/secure-endpoint',
        'headers': {'Authorization': 'Bearer <token>'},
        'body': {'data': 'example'}
    }
